web/modules/custom/dpl_react/dpl_react.libraries.yml

```diff
@@ -239,6 +239,21 @@ material-grid-manual:
     - dpl_react/base
     - dpl_react/handler
 
+material-search:
+  remote: https://github.com/danskernesdigitalebibliotek/dpl-react
+  license:
+    name: GNU AFFERO
+    url: https://github.com/danskernesdigitalebibliotek/dpl-react/blob/master/LICENSE
+    gpl-compatible: true
+  js:
+    /libraries/dpl-react/MaterialSearch.js: {}
+  css:
+    base:
+      /themes/custom/novel/assets/dpl-design-system/css/admin-base.css: {}
+  dependencies:
+    - dpl_react/base
+    - dpl_react/handler
```

Comment:
Please move `admin-base.css` to a separate library and then include it as a dependency like we do for `dpl_react/base`.